util/util.go

```diff
@@ -173,8 +173,7 @@ func GetServiceIDByPostmanCollectionID(c rest.FrontClient, ctx context.Context,
 func GetOrCreateServiceIDByPostmanCollectionID(c rest.FrontClient, collectionID string) (akid.ServiceID, error) {
 	// Normalize the collectionID.
 	collectionID = strings.ToLower(collectionID)
-	unexpectedErrMsg := "Something went wrong while starting the Agent. " +
-		"Please contact Postman support (" + consts.SupportEmail + ") with the error details"
+	unexpectedErrMsg := "Something went wrong while starting the agent."
```

Comment:
This is fine, because the collection path is deprecated.

However, the point I was trying to make in the bug is that if the problem is easily fixable by the user we should recommend such a fix first, not recommend contacting Postman support immediately.